sale_triple_discount/tests/test_sale_triple_discount.py

```diff
@@ -3,49 +3,47 @@
 # Copyright 2022 Manuel Regidor - Sygel Technology
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-import json
 
 from odoo.tests import common
 
 
 class TestSaleOrder(common.TransactionCase):
-    @classmethod
-    def setUpClass(cls):
-        super(TestSaleOrder, cls).setUpClass()
-        cls.partner = cls.env["res.partner"].create({"name": "Mr. Odoo"})
-        cls.product1 = cls.env["product.product"].create(
+    def setUp(self):
```

Comment:
Why do you change from `setUpClass` to `setUp`  It's better to create all the required stuff into setUpclass than in stUp since setUpClass is executed only once for all the test (with a rollback to savepoint after each test) bu setUp is executed bedore each test...